src/SvelteMarkdown.svelte

```diff
@@ -18,7 +19,10 @@
   $: slugger = source ? new Slugger : undefined
   $: combinedOptions = { ...defaultOptions, ...options }
   $: {
-    lexer = new Lexer(combinedOptions)
+    marked.setOptions(combinedOptions)
+    marked.use(use)
+
+    lexer = new Lexer()
```

Comment:
Since we're using all of marked now, is there a difference on build size between importing `Lexer` vs doing `new marked.Lexer()`?

Reply:
I did some rudimentary tests with the svelte template where I added a SvelteMarkdown component with an extension (as in the first comment of this PR):
There were *no differences* in files size (after `yarn build`) when using it as is VS removing the Lexer/Slugger imports entirely and writing
```javascript
slugger = source ? new marked.Slugger : undefined
```
and
```javascript
tokens = isInline ? marked.Lexer.lexInline(source) : marked.Lexer.lex(source)
```
Removing the imports seems to have no effect on build size (and could just be done for cosmetic reasons).